client/app/core/navigation.config.js

```diff
@@ -3,42 +3,43 @@
 /** @ngInject */
 export function navConfig(NavigationProvider) {
   const dashboard = createItem(
-    N_('Dashboard'),
+    'Dashboard',
     'dashboard',
     'fa fa-dashboard'
   );
+
   const services = createItem(
-    N_('Services'),
```

Comment:
Translation happens in the navigation controller.  It was already doing it there as well.  It is better in this instance to do it in the navigation controller.